server/utility/admin_emails.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+from girder.constants import AccessType
+from girder.utility.model_importer import ModelImporter
+
+
+def _getUserEmail(user):
+    """Convenience function to get the email field from a user document."""
+    return user['email']
+
+
+def _getUser(userId):
+    """Convenience function to get a user document from a user ID."""
+    user = ModelImporter.model('user').load(userId, force=True)
+    return user
+
+
+def _getAdminEmails(acl):
+    """
+    Given an access list, return a list of the email address for admin users in
+    the access list.
+
+    :param acl: an access list such as that returned by
+        AccessControlledModel.getFullAccessList()
+    :type acl: dict
+    """
+    users = acl.get('users', None)
+    adminUserIds = [user['id']
+                    for user in users
+                    if user['level'] >= AccessType.ADMIN]
+    adminUsers = map(_getUser, adminUserIds)
```

Comment:
You could just use ``lambda user: user['email']`` (or ``operator.itemgetter('keys')`` if you really care about performance) as the function argument. Making a whole new method (even a private one) to get an obviously-named dict entry seems a little overkill.